src/ui/components/CardsPlaceholder/CardsPlaceholder.tsx

```diff
@@ -21,6 +21,7 @@ const CardsPlaceholder = ({
         shape="round"
         expand="block"
         className="primary-button"
+        data-testid={`${testId}-button`}
```

Comment:
Since I am currently refactoring the Identifiers tab I would advise to wait before adding tests for this particular tab as things may change quite significantly. For example this button is now replaced with the new `PageFooter` component etc. Below here is an example of what to expect for both Identifiers and credentials tabs. As I anticipated yesterday, I believe this PR should be for "verify seed phrase and create password" so maybe you could test the elements in the Identifiers tab at a later stage after I complete the refactor?

![19122023120956](https://github.com/cardano-foundation/cf-identity-wallet/assets/119612231/485b03c1-9745-473d-a311-70c70b27936e)

Reply:
ok

Reply:
@obstar The rule says we can comment out code if we explain why :) 

I have completed the refactor of this page yesterday (that was why you found all those merging issues you have just fixed) so feel free to bring it back if that makes sense to you.

Reply:
@sdisalvo-crd fix added